src/core/server/rendering/views/template.tsx

```diff
@@ -95,6 +95,33 @@ export const Template: FunctionComponent<Props> = ({
       />
     </svg>
   );
+
+  const renderEnabledOrDisabledLoadingBar = () => {
+    if (!injectedMetadata.branding.loadingLogoUrl && injectedMetadata.branding.smallLogoUrl) {
+      return <div className="osdProgress" />;
+    }
+  };
+
+  const renderBrandingEnabledOrDisabledLoadingLogo = () => {
```

Comment:
Two functions name renderEnabledOrDisabledLoadingBar and renderBrandingEnabledOrDisabledLoadingLogo seems not match. Maybe:
1) renderEnabledOrDisabledLoadingBar --> renderBrandingEnabledOrDisabledLoadingBar to match the second one
or 
2) renderBrandingEnabledOrDisabledLoadingLogo --> renderEnabledOrDisabledLoadingLogo to match the first function

Reply:
Thanks! Yea i made it consistent by changing renderEnabledOrDisabledLoadingBar to renderBrandingEnabledOrDisabledLoadingBar.

Reply:
thx